app/controllers/provider/admin/dashboards_controller.rb

```diff
@@ -16,6 +16,9 @@ def show
     @services           = current_user.accessible_services
     @messages_presenter = current_presenter
     @unread_messages_presenter = unread_messages_presenter
+    return unless flash[:quick_starts_notice]
+
+    flash[:quick_starts_notice] = t('provider.admin.dashboards.quick_starts_html', link: provider_admin_quickstarts_path).html_safe
```

Comment:
In order to make it easier to understand that we are using one flash object to transfer data between controllers and the other to show the notice, I would call the objects `flash[:first_login]` and `flash[:notice]`

```suggestion
    return unless flash[:first_login]

    flash[:notice] = t('provider.admin.dashboards.quick_starts_html', link: provider_admin_quickstarts_path).html_safe
```